packages/js/package.json

```diff
@@ -46,6 +46,7 @@
     "babel-plugin-macros": "^2.8.0",
     "babel-plugin-transform-typescript-metadata": "^0.3.1",
     "chalk": "^4.1.0",
+    "glob": "7.1.4",
     "fast-glob": "3.2.7",
```

Comment:
Would now be a good time to sawp these from glob -> fast-glob? I know this isn't a particularly perf sensitive codepath, but having both installed under js looks a bit silly.

Reply:
The imports are part of the file rename

Reply:
Perhaps not now as this Pr is mainly just to move the assets utils from workspace, a further PR can be made to focus on improving the perf

Reply:
Unless the APIs are similar, in which I could try?

Reply:
The assets file didnt have associated test files, so where could I confirm that it is functioning correctly if i was to refactor it?

Reply:
The APIs are identical IIRC, I'm not sure what all uses these utils still though, you'd have to look for usages and check manually (or write the tests)

Reply:
swc + tsc in the js package i believe. I'll check and make the change

Reply:
swc + tsc use the copyAssetsHandler that's already in the JS package

Reply:
Yeah but they also use what's in workspace: https://github.com/nrwl/nx/blob/ebdb315a3a416e4ba44a0b8b69462243a3d71094/packages/js/src/executors/swc/swc.impl.ts#L56

I've made the change anyway for those utils to use fast-glob.

Reply:
@AgentEnder can you double check that this is ok?

Reply:
Yeah - I'll look at it now :)